### ISSUE TITLE
Update-OctopusVariableSet crashing when adding environment

### ISSUE THREAD

**Issue by magnustomte** [Dalmirog/OctoPosh#185]
When running Update-OctopusVariableSet with an environment specified in the json-file the following error occurs:

```
Unable to find type [Octopus.platform.Model.Scopefield].
At C:\Program Files\WindowsPowerShell\Modules\octoposh\0.4.6\scripts\Update-OctopusVariableSet.ps1:97 char:25
+ ...             $newvar.Scope.Add([Octopus.platform.Model.Scopefield]::$s ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (Octopus.platform.Model.Scopefield:TypeName) [], RuntimeException
    + FullyQualifiedErrorId : TypeNotFound
```

[Octopus.platform.Model.Scopefield] should perhaps be [Octopus.Client.Model.ScopeField]?

**Comment by Dalmirog** [comment]
This was fixed in `0.5.1` thanks to a PR from someone in the community: https://github.com/Dalmirog/OctoPosh/commit/780eddee25a12389453e80da47ee863c78a39b45

Can you please try in that version?

**Comment by Dalmirog** [comment]
Closing Issue but willing to open it again if the issue persists. 

You can also contact me @ https://gitter.im/Dalmirog/OctoPosh for questions.
